src/backend/renderer.js

```diff
@@ -1661,19 +1692,59 @@ export function attach(
     startProfiling();
   }
 
+  // React will switch between these implementations depending on whether
+  // we have any manually suspended Fibers or not.
+
+  function shouldSuspendFiberAlwaysFalse() {
+    return false;
+  }
+
+  let forceFallbackForSuspenseIDs = new Set();
+  function shouldSuspendFiberAccordingToSet(fiber) {
+    const id = getFiberID(getPrimaryFiber(((fiber: any): Fiber)));
+    return forceFallbackForSuspenseIDs.has(id);
+  }
+
+  function overrideSuspense(id, forceFallback) {
+    if (
+      typeof setSuspenseHandler !== 'function' ||
+      typeof scheduleUpdate !== 'function'
```

Comment:
nit could we just check `!supportsEditingSuspense` here?

Reply:
Flow wouldn’t be happy

Reply:
Oh, right. lol. Ok